packages/cli/src/commands/generate/deploy/providers/aws_serverless.js

```diff
@@ -0,0 +1,98 @@
+import path from 'path'
+import fs from 'fs'
+
+import { getPaths } from 'src/lib'
+
+export const apiProxyPath = '/api'
```

Comment:
```suggestion
```

We've defaulted to something else now: `/.redwood/` because we're worried that a user might have an `api` route. I think it's fine to leave this out and keep it at the standard location.